lib/sockopt.c

```diff
@@ -72,6 +72,21 @@ int getsockopt_so_sendbuf(const int sock)
 	return optval;
 }
 
+int getsockopt_so_recvbuf(const int sock)
+{
+	uint32_t optval;
+	socklen_t optlen = sizeof(optval);
+	int ret = getsockopt(sock, SOL_SOCKET, SO_RCVBUF, (char *)&optval,
+			     &optlen);
+	if (ret < 0) {
+		flog_err_sys(EC_LIB_SYSTEM_CALL,
+			     "fd %d: can't getsockopt SO_RECVBUF: %d (%s)", sock,
```

Comment:
as Quentin says "Hate to do this", but it should be SO_RCVBUF :smile:

Reply:
fixed

Reply:
sorry, I still see `SO_RECVBUF` :/ maybe missed force push?